news/2024/2024-09-17-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,168 @@
+
+### Rhythm complexity improvements
+
+A [change](https://github.com/ppy/osu/pull/28871) proposed by [StanR](https://osu.ppy.sh/users/7217455) fixes some flaws in the rhythm complexity calculations.
+
+As a reminder of how rhythm works, notes are grouped into 'islands' (for example, a triple pattern is described as a 3 note island) and complexity is assessed using these islands.
+
+The first improvement in this change is removing a size bonus on island length and uncapping the island size. Previously, there was a bonus applied based on island length sizes that would cause long bursts to gain a lot of rhythm complexity even if they were not complex.
+
+The second improvement in this change heavily nerfs repeated islands. This causes maps which abuse the spamming of triples to increase pp (one example is [SLoWMoTIoN](https://osu.ppy.sh/beatmapsets/295848#osu/988793)) to have a considerable nerf:
+
+![](/wiki/shared/news/2024-09-17-performance-points-star-rating-updates/island-repetition-nerf.png)
+
+The third improvement in this change is a general buff to the initial rhythm bonus. This helps maps with high rhythm complexity such as [Asteroid Field of DECAPLETS](https://osu.ppy.sh/beatmapsets/1666685#osu/3403124) to gain a small bonus.
+
+The final improvement in this change is a nerf to rhythm complexity based on how "doubletappable" an island is. "doubletapness" is assessed by comparing the 300 hit window size to the time between the current and previous object and deciding if the player has enough time to comfortably double-tap. This doubletap nerf already existed in speed, with this change now bringing it over to rhythm complexity too.
+
+### Speed distance hotfix
+
+A [change](https://github.com/ppy/osu/pull/29980) proposed by [tsunyoku](https://osu.ppy.sh/users/11315329) was created to hotfix the speed distance bonus.
+
+The distance bonus exists to reward flow aim by multiplying it onto the speed difficulty. In this hotfix, rather than the bonuses being multiplicative they have been changed to be additive to overall decrease the effect the distance bonus has on the speed value.
+
+Alongside this, the distance bonus has a change in scaling that results in lower spacing flow aim to be less rewarded.
+
+![](/wiki/shared/news/2024-09-17-performance-points-star-rating-updates/distance-bonus.png)
+
+Finally, a new multiplier was added to the final result of the distance bonus to decrease it overall.
+
+### Sliders now contribute to accuracy pp
+
+A [change](https://github.com/ppy/osu/pull/27063) submitted by [tsunyoku](https://osu.ppy.sh/users/11315329) has been created to include the slider count in accuracy pp if a score was set using slider head accuracy.
+
+Accuracy pp uses its own formula to create a separate object count that has always excluded sliders as traditionally, osu! never had accuracy on slider heads. Slider head accuracy is now a possibility in osu!(lazer), so accuracy pp has been adjusted to include sliders.
```

Comment:
```suggestion
Traditionally, osu! has not had accuracy on slider heads, and so Accuracy pp traditionally excludes sliders in its calculation. However, osu!(lazer) has slider head accuracy, and so Accuracy pp now includes sliders.
```

Reply:
this is better but I realise this is missing clarification that it'll only include sliders if you're playing with slider head accuracy

Reply:
fixed in https://github.com/ppy/osu-wiki/pull/12159/commits/35f30dba96b813c58be1c4364103732d64a4c3dd